src/main/java/seedu/address/logic/commands/results/CommandResult.java

```diff
@@ -78,10 +78,10 @@ public boolean equals(Object other) {
 
         CommandResult otherCommandResult = (CommandResult) other;
         return feedbackToUser.equals(otherCommandResult.feedbackToUser)
-                && isShowPreview == otherCommandResult.isShowPreview
-                && isGenerate == otherCommandResult.isGenerate
-                && isShowHelp == otherCommandResult.isShowHelp
-                && isExit == otherCommandResult.isExit;
+            && isShowPreview == otherCommandResult.isShowPreview
```

Comment:
I thought this would throw off checkstyle? The initial format wasn't supposed to throw off checkstyle right?